### PR TITLE
Psproduct-3993 new tips version

### PR DESCRIPTION
**Pull request by abetsxsolla**

### CODE REVIEW

File: example/index.html [hunk 1]
```diff
@@ -18,30 +18,13 @@
...
-
-    <div id="widget-example-compact">
-    </div>
-
-    <br />
-
-    <div id="widget-example-full">
-    </div>
-
-    <div class="box">
```
Comment (sheff146) [inline example/index.html]: I think it is better to leave this text.

File: example/index.html [hunk 2]
```diff
@@ -84,22 +67,12 @@
...
-                target_element: '#widget-example-full'
-            }, options));
-
-            var compactWidget = XPay2PlayWidget.create(_.extend({
-                template: 'compact',
-                target_element: '#widget-example-compact'
-            }, options));
-
             var tinyWidget = XPay2PlayWidget.create(_.extend({
                 template: 'tiny',
```
Comment (sheff146) [inline example/index.html]: This parameter is redundant now

File: example/index.html [hunk 3]
```diff
@@ -84,22 +67,12 @@
...
                 target_element: '#widget-example-tiny'
```
Comment (sheff146) [inline example/index.html]: "tiny" part is redundant.

File: src/styles/base/widget.scss [hunk 4]
```diff
@@ -31,6 +31,18 @@ $widget-class: '.' + $widget-prefix;
     @include error-message;
   }
 
+  &-button-block{
+    position: absolute;
+    left: 0px;
+    right: 0px;
+    bottom: 0px;
+    display: flex;
```
Comment (sheff146) [inline src/styles/base/widget.scss]: 👍

File: src/views/payment-button.jsx [hunk 5]
```diff
@@ -40,75 +16,38 @@ var PaymentButton = React.createClass({
...
                 </div>
             );
 
-        var tipsList = this.state.isTipsListOpened && (
-                <TipsList baseClassName={this.props.baseClassName} tips={this.props.tips} onSelect={this.onTipSelect}/>
-            );
-
         return (
-            <div className={this.props.baseClassName + '-payment-button'}
+            <div className={this.props.baseClassName + '-payment-button ' +
```
Comment (sheff146) [inline src/views/payment-button.jsx]: Change tag name to "button"

File: example/index.html (hunk 1 above)
Comment (abetsxsolla) [inline example/index.html]: fix

**Pull request merged**